apps/web/lib/features/team-members-kanban-view.tsx

```diff
@@ -101,7 +156,13 @@ export const KanbanView = ({ kanbanBoardTasks }: { kanbanBoardTasks: IKanban })
 
 		return selectState[0].color;
 	};
+	function reorderColumn(column: any[], sourceIndex: number, destinationIndex: number) {
```

Comment:
@Anishali2 I see now you rewrite `reorderColumn` but you used "function" instead of "arrow" syntax I think we use everywhere, any reason? Can we try to be consistent and use arrow functions definitions etc?